duckduckgo/iOS
# iOS Voice Search - Custom Dialog & Experiment

Task/Issue URL: https://app.asana.com/0/72649045549333/1200934152050177/f

**Description**:
This PR changes 4 things:
* Removes experiment related code. i.e: Feature will be released to everyone
* Removes custom UIAlert dialog that was displayed before the system dialog
* Moves all the SFSpeechRecognizer availability check to a background queue
* Keeps the UI updated depending on the SFSpeechRecognizer availability callbacks by it's delegate

**Steps to test this PR**:
1. On a fresh install, tap on the microphone button
2. You should only see the system dialog asking for mic permission

------
1. Tap on the mic button on the Omnibar and perform a voice search (please use a real device)
2. Check if the voice search was triggered correctly 

------
1. Check that there are no experiment condition being applied for this feature
-----

1. Check that the microphone button is *not* available on iOS 14 or lower
-----
Original PR for context: https://github.com/duckduckgo/iOS/pull/1003


**Device Testing**:

* [x] iPhone 6S

**OS Testing**:


**Theme Testing**:
